ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Background.hs

```diff
@@ -255,6 +258,10 @@ copyAndSnapshotRunner cdb@CDB{..} gcSchedule replayed =
     SecurityParam k      = configSecurityParam cdbTopLevelConfig
     LgrDB.DiskPolicy{..} = LgrDB.getDiskPolicy cdbLgrDB
 
+    getSlotNo :: ExtLedgerState blk -> WithOrigin SlotNo
```

Comment:
you mean 

```
ledgerTipSlot ::
     forall blk. UpdateLedger blk
  => LedgerState blk -> WithOrigin SlotNo
ledgerTipSlot = pointSlot . (ledgerTipPoint (Proxy @blk))
```

?

Reply:
I mixed up `Ouroboros.Consensus.Ledger.Basics` and `.Abstract`. My suggestion is actually to add the `.Basics` import in order to bring in `getTipSlot`.

(Apparently those two modules are separate to avoid a cyclic dependency -- I haven't yet investigated how to simplify that.)